client/scripts/boinc-client.service.in

```diff
@@ -5,6 +5,19 @@ After=network-online.target
 
 [Service]
 ProtectHome=true
+PrivateTmp=true
+ProtectSystem=strict
+NoNewPrivileges=true
+ProtectKernelModules=true
+ProtectKernelTunables=true
+ProtectControlGroups=true
+RestrictRealtime=true
+RestrictAddressFamilies=AF_INET AF_INET6 AF_UNIX
+RestrictNamespaces=true
+PrivateUsers=true
+CapabilityBoundingSet=
+ProtectControlGroups=true
```

Comment:
duplicate

Reply:
Sorry about that. I just forced push the removal of that duplication.